src/components/GlobalLinterMessages/index.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+
+import { LinterMessage as LinterMessageType } from '../../reducers/linter';
+import LinterMessage from '../LinterMessage';
+import { getCodeLineAnchorID } from '../CodeView/utils';
+
+export type PublicProps = {
+  className?: string;
+  messages?: LinterMessageType[] | null | void;
```

Comment:
This property should not be optional, i.e. you can remove the question mark. This is because the caller would be mistaken to omit the property, like `<GlobalLinterMessages />`. We want TypeScript to tell them that they forgot to specify `messages`.